docs/index.rst

```diff
@@ -0,0 +1,12 @@
+python-cloudant documentation
+=============================
+
+This is the official Cloudant client library for Python.
+
+.. toctree::
+   :maxdepth: 4
```

Comment:
There is a [generated index](http://python-cloudant.readthedocs.org/en/53377-docs-via-sphinx/genindex.html).  The link to it is at the bottom of the main doc page.  I believe that the maxdepth level refers to TOC display in the main browser and navigation frames only and not the generated index.  I suppose proof of that is that the generated index has method references.  Incidently, the value of 4 was auto generated when I ran the API reference creation script provided by Sphinx.  It appears that 4 seems to be the "optimal" value for display of TOC in both the main and navigation frames.